torch_tvm/compiler.h

```diff
@@ -12,17 +12,29 @@
 
 #include "memory_utils.h"
 
+struct TVMGraphInputInfo {
+  TVMGraphInputInfo(bool is_param_, std::string tvm_var_name_) {
```

Comment:
Any reason this can’t just be a tuple?

Reply:
It makes code less readable. I dont think doing std::get<0,1> etc, helps understand what the parameters are.

Reply:
I did think about this fwiw.

Reply:
string && ?

Reply:
@yinghai, there is a constructor with string&&. A separate one.